model/src/test/kotlin/JsonSchemaTest.kt

```diff
@@ -73,6 +73,19 @@ class JsonSchemaTest : StringSpec() {
 
             errors should beEmpty()
         }
+
+        "package-configuration.yml validates" {
```

Comment:
Nit: All other tests say "validates successfully", so we should probably do so here, too.